src/OpenLoco/src/Interop/Hooks.cpp

```diff
@@ -805,7 +805,7 @@ void OpenLoco::Interop::registerHooks()
 
     registerHook(
         0x004392BD,
-        [](registers& regs) FORCE_ALIGN_ARG_POINTER -> uint8_t {
+        []([[maybe_unused]] registers& regs) FORCE_ALIGN_ARG_POINTER -> uint8_t {
```

Comment:
can't we just have these unnamed the hooks aren't going to change apart from be deleted.

Reply:
True, i'll change it.